packages/extension/web/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "glow",
```

Comment:
To match the manifest, we should also name this to "firefly-web" or whatever you think is best.

Reply:
ok, renamed to `firefly-extension`

Reply:
Thank you :pray: